src/wp-admin/css/admin-menu.css

```diff
@@ -63,6 +63,7 @@
 	position: relative;
 	float: left;
 	z-index: 9990;
+	outline: 1px solid transparent;
```

Comment:
This element is not guaranteed to be 100% tall:

![adminmenuwrap](https://github.com/user-attachments/assets/fde7cb39-3b4f-427d-8f96-f0db0e8cf46f)

I think it would be a good idea to apply an outline to the `#adminmenuback`.

Reply:
Sure @t-hamano, thanks for this. I have thought of this, but then realized it may look odd to not add 100%, hence added it on `#adminmenuwrap`. I would add this change.